.github/workflows/hil_sample_zephyr.yml

```
@@ -226,6 +241,17 @@ jobs:
           elif [[ "${hil_board}" == "mimxrt1024_evk" ]]; then
             JLinkExe -nogui 1 -if swd -speed auto -device MIMXRT1024XXX5A -CommanderScript erase_mimxrt1024_evk.jlink -SelectEmuBySN ${!SNR_VAR}
 
+          elif [[ "${hil_board}" == "rak5010_nrf52840" ]]; then
+            gdb \
+            -ex "set confirm off" \
+            -ex "target extended-remote "${BLACKMAGIC_PORT} \
```

```suggestion
            -ex "target extended-remote "${CI_BLACKMAGICPROBE_PORT} \
```

Update to match what self-hosted runner board recogntiion script generates.